src/cascadia/WindowsTerminal/WindowsTerminal.vcxproj

```diff
@@ -152,6 +145,15 @@
       <Output TaskParameter="TargetOutputs" ItemName="_PackagingOutputsFromOtherProjects"/>
     </MSBuild>
 
+    <!-- **BEGIN VC LIBS HACK** -->

```

Comment:
Why did this move down? Is there something above here that it needs done first? Leave comment explaining that?

Reply:
I wanted to toe the line between explaining “I moved the wildcards out of the root item groups and into the targets” every time I moved a wildcard out of the root item groups and into a target, but I can leave a comment on this one.